### PR TITLE
Hebrew translation

### PR DESCRIPTION
**Pull request by tom-programming**
Hi!

Your work is amazing. It's the only HTML/Javascript editor where it is really easy to add tables. I now also added a Hebrew translation. However Hebrew is RTL and it would be good to have a language-related flag that sets the default text direction for a language.

### CODE REVIEW

File: src/lang/he.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                url: 'כתובת קשור',
+                text: 'תיאור',
+                newWindowCheck: 'פתח בחלון חדש',
+                downloadLinkCheck: 'קישור להורדה',
+                bookmark: 'סמניה'
+            },
+            mathBox: {
+                title: 'נוסחה',
+                inputLabel: 'סימנים מתמטים',
+                fontSizeLabel: 'גודל דופן',
```
Comment (tom-programming) [inline src/lang/he.js:92]: fontSizeLabel should be
גודל גופן

it was a typo

File: src/lang/he.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+            undo: 'בטל',
+            redo: 'חזור',
+            preview: 'תצוגה מקדימה',
+            print: 'הדפס',
+            tag_p: 'פסקה',
+            tag_div: 'רגילה (DIV)',
+            tag_h: 'כותרת',
+            tag_blockquote: 'ציטוט',
+            tag_pre: 'קוד',
+            template: 'תבניט',
```
Comment (tom-programming) [inline src/lang/he.js:73]: template should be
תבנית

sorry, another typo

**Comment by tom-programming** [comment]
I reviewed my own work and found two typos. I used the comment feature because I am not familiar with post-PR corrections. Sorry for the errors

**Comment by JiHong88** [comment]
@tom-programming Thank you for your contribution.!!
I'll notify you after version update.
Thank you :)

**Comment by JiHong88** [comment]
The [2.36.2](https://github.com/JiHong88/SunEditor/releases/tag/2.36.2) version has been updated.
Thank you.

**Pull request merged**